src/OMSimulatorLib/Component.h

```diff
@@ -101,6 +101,8 @@ namespace oms
     virtual oms_status_enu_t setInteger(const ComRef& cref, int value) {return logError_NotImplemented;}
     virtual oms_status_enu_t setReal(const ComRef& cref, double value) {return logError_NotImplemented;}
 
+    virtual oms_status_enu_t deleteStartValues(const ComRef& cref) {return oms_status_ok;}
```

Comment:
This is just deleting a single start value at a time, right? Hence I would rename it to `deleteStartValue`.

Reply:
renamed the function name to deleteStartValue see https://github.com/OpenModelica/OMSimulator/pull/758/commits/ee8d2f44ed637c3fcc93e87b8f81dbffd999b5ff